bpm/bonita-util/bonita-common-util/src/test/java/org/bonitasoft/engine/bdm/BusinessObjectModelConverterTest.java

```diff
@@ -80,10 +79,25 @@ public void unzipADifferentZipThrowAnException() throws Exception {
 
     @Test
     public void should_be_backward_compatible() throws Exception {
```

Comment:
we may also add another backward test where xml file contains new attibutes to ensure they are not overrided with default values